frontend/apps/docs/content/docs/community-resources.mdx

```diff
@@ -8,10 +8,17 @@ description: A collection of community-created resources and articles about Liam
 ### MySQL
 
 - [Liam ERD で tbls からサクッと MySQL の ER 図を作成してみた](https://zenn.dev/nextbeat/articles/liam-erd-tbls-mysql)
-  - February 2024 • zenn.dev • [translated](https://zenn-dev.translate.goog/nextbeat/articles/liam-erd-tbls-mysql?_x_tr_sl=auto&_x_tr_tl=ja&_x_tr_hl=ja&_x_tr_pto=wapp&_x_tr_hist=true)
+  - February 2024 • zenn.dev • [translated](https://zenn-dev.translate.goog/nextbeat/articles/liam-erd-tbls-mysql?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp&_x_tr_hist=true)
   - A guide to generating MySQL ER diagrams using Liam ERD with tbls
   - Includes practical examples using the sakila sample database
 
+### Ruby on Rails
+
+- [ER図自動生成のためにLiam ERDを試験導入してみました！](https://tech-gree-x-com.translate.goog/automated-er-diagram-generation/index.html)
```

Comment:
I got an error when accessing this URL.
In this line, maybe it's better to just use the original Japanese article URL.

Reply:
Exactly, thanks for noticing!

```suggestion
- [ER図自動生成のためにLiam ERDを試験導入してみました！](https://tech.gree-x.com/automated-er-diagram-generation/index.html)
```